src/demo-app/components/icon-demo/icon-demo.html

```diff
@@ -110,3 +110,13 @@ <h3 class="demo-content__headline">Using Font Awesome w/size modifier</h3>
     <mdc-icon #fasize fontSet="fa" fontIcon="fa-map fa-4x"></mdc-icon>
   </div>
 </div>
+
+<h2>Font Awesome 5</h2>
```

Comment:
@rapropos This has me thinking.. 🤔 

We should implement a service for registering a font's CSS class. 

Set a default `fontSet` class
```ts
mdcIconRegistry.defaultFontSetClass('fa');
```

Making Font Awesome usage with `mdc-icon` easy throughout app
```html
<mdc-icon>cakeIcon</mdc-icon>
```

Reply:
That is one option, but at least personally I ran into several counterintuitive problems with FA5's SVG magic in an Angular environment. For example, structural directives such as `*ngIf` and dynamic binding of classes to change icons simply don't work on `<i>` elements. The `<fa-icon>` directive in angular-fontawesome smooths over all of these problems, allowing one to use FA5 in a much more idiomatic Angular way.

I considered submitting a PR using `fa-icon` under the hood in `mdc-icon`, but after thinking a bit about it, it seemed out of scope for angular-mdc-web, and once I discovered that content projection worked without modifying the existing angular-mdc-web codebase, I decided that I thought simply documenting that fact and illustrating how to use it with angular-fontawesome was the most prudent course of action.

Reply:
Hey, @rapropos I plan to revisit PR after finishing v0.36.0-0 update. I'll need some free time to review FA5.

Reply:
Turns out v0.36.0-0 required some `mdc-icon` improvements..

New public UX state methods for PR https://github.com/trimox/angular-mdc-web/pull/1021